archetype/engine-v2/src/main/java/io/helidon/build/archetype/engine/v2/ast/Model.java

```diff
@@ -264,6 +266,15 @@ public String template() {
             return template;
         }
 
+        /**
+         * Test if this value can be overridden.
```

Comment:
The meaning is that the value is an override itself and not that it can be overridden right ?